<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UrT PyMonitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="serverList">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>506</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionAdd_Server"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Monitoring"/>
    <addaction name="actionStop_Monitoring"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Server">
   <property name="text">
    <string>Add Server</string>
   </property>
  </action>
  <action name="actionStart_Monitoring">
   <property name="text">
    <string>Start Monitoring</string>
   </property>
  </action>
  <action name="actionStop_Monitoring">
   <property name="text">
    <string>Stop Monitoring</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
